services/manage/app/services/user.js

```diff
@@ -7,4 +7,6 @@ export default Service.extend(ObjectQueryManager, {
 
   model: computed.alias('session.data.authenticated.user'),
   isAuthenticated: computed.reads('session.isAuthenticated'),
+  profileShown: false,
```

Comment:
Is there a way to shore some of this up - perhaps with different semantics (such as `hasValidUserData` or something?). `profileShown` seems to vague (it's hard for me to understand what it means)